recipes/default.rb

```diff
@@ -19,6 +19,8 @@
 # limitations under the License.
 #
 
+include_recipe 'ntp::default' unless node['krb5']['include_ntp'].to_s == 'false'
```

Comment:
We do this in a lot of places. This allows us to support either case where the user of the cookbook uses a string or a boolean.

`node['krb5']['include_ntp'] = false`
or
`node['krb5']['include_ntp'] = 'false'`